### DISCUSSION TITLE
Unable to remove Adobe DRM using v10.0.3

### DISCUSSION THREAD

**Discussion by Andrewboker** [noDRM/DeDRM_tools#389] [Q&A]
Hi. I bought a book from thalia.de and I need to put it on my kindle.

I have tried using Calibre 6.24 and deDRM v10.0.3 on a mac running Ventura 13.4.1

I get the following log error - is anyone able to help me please?

`Using calibre Qt style: True
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
ApplicationPaletteChange event received
calibre Debug log
calibre 6.24  embedded-python: True
macOS-13.4.1-arm64-arm-64bit Darwin ('64bit', '')
('Darwin', '22.5.0', 'Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:19 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T8103')
Python 3.10.1
OSX: ('13.4.1', ('', '', ''), 'arm64')
Interface language: en_GB
Successfully initialized third party plugins: DeDRM (10, 0, 3)
calibre 6.24  embedded-python: True
macOS-13.4.1-arm64-arm-64bit Darwin ('64bit', '')
('Darwin', '22.5.0', 'Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:19 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T8103')
Python 3.10.1
OSX: ('13.4.1', ('', '', ''), 'arm64')
Interface language: en_GB
Successfully initialized third party plugins: DeDRM (10, 0, 3)
QPA platform: cocoa
devicePixelRatio: 2.0
logicalDpi: 72.0 x 72.0
physicalDpi: 127.50000191485789 x 127.50000191485789
[0.00] Starting up...
[0.02] Showing splash screen...
[0.18] splash screen shown
[0.18] Initializing db...
[0.19] db initialized
[0.19] Constructing main UI...
[0.51] main UI initialized...
[0.51] Hiding splash screen
Starting QuickView
2023-08-14 13:25:24.154 calibre-debug[13266:1884248] +[CATransaction synchronize] called within transaction
DeDRM v10.0.3: Trying to decrypt The Spear Cuts Through Water.epub
DeDRM v10.0.3: Verifying zip archive integrity
DeDRM v10.0.3: The Spear Cuts Through Water.epub is a secure Adobe Adept ePub for UUID ea2338d7-dbcb-49ec-9904-ae74ad5bb8d7
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "encodings/hex_codec.py", line 19, in hex_decode
ValueError: string argument should contain only ASCII characters

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "calibre/customize/ui.py", line 187, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 1067, in run
    decrypted_ebook = self.ePubDecrypt(path_to_ebook)
  File "calibre_plugins.dedrm.__init__", line 533, in ePubDecrypt
    userkey = codecs.decode(userkeyhex, 'hex')
ValueError: decoding with 'hex' codec failed (ValueError: string argument should contain only ASCII characters)
[4.60] splash screen hidden
[4.60] Started up in 4.60 seconds with 2 books
Added The Spear Cuts Through Water: A Novel to db in: 0.0
Added 1 books in 0.3 seconds
`

**Comment by ElleKayEm** [comment]
Answered in the issue you posted #388
